keymap.cpp

```diff
@@ -0,0 +1,304 @@
+#include <linux/keyboard.h>
+#include <Fcitx5/Utils/fcitx-utils/keysymgen.h>
```

Comment:
This is wrong, you should use find_package(Fcitx5Utils), link to Fcitx5::Utils and include <fcitx-utils/keysym.h> instead.